src/build.d

```diff
@@ -601,10 +601,10 @@ alias dmdPGO = makeRule!((builder, rule) {
         .condition(() => PGOState.checkPGO(dmdKind))
         .deps([finalDataMerge])
         .commandFunction({
-            auto addArgs = pgoState.pgoUseFlags ~ "-wi";
-            auto cmd = [env["HOST_DMD_RUN"], "-run", "src/build.d", "ENABLE_RELEASE=1",
-            "ENABLE_LTO=1",
-            "DFLAGS="~joiner(addArgs, " ").to!string, "--force", "-j"~jobs.to!string];
+            const extraFlags = pgoState.pgoUseFlags ~ "-wi";
+            const scope cmd = [thisExePath, "HOST_DMD="~env["HOST_DMD_RUN"],
```

Comment:
Adding `HOST_DMD` just to make sure (or in the hopes of) making sure the 2nd build uses the same host compiler.

Reply:
I'd have thought this is a non-issue but doesn't hurt to be explicit.